packages/@angular/cli/lib/config/schema.json

```diff
@@ -514,6 +514,28 @@
           "default": true
         }
       }
+    },
+    "update": {
+      "type": "object",
+      "description": "Configuration for CLI update checks.",
+      "default": {},
+      "properties": {
+        "lastChecked": {
+          "type": "object",
+          "default": {},
+          "additionalProperties": true
+        },
+        "checkFrequency": {
+          "type": "string",
+          "description": "Defines how often to check for an updated version.",
+          "default": "1d"
```

Comment:
We don't support it, but there's a way for validating strings against regexes in JSON Schema. That would be nice to have here for at least validators and IDEs that support it, and maybe we will soon :)

See http://json-schema.org/latest/json-schema-validation.html#rfc.section.6.8